UI/bucketlist.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>My Bucket List</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/css/materialize.min.css">
```

Comment:
I love the way you have indented this header.

**_I don't know if this applies as good practice_** but I would recommend you download the `materialize.min.css` file to reduce the rendering time of your interface and just in case the internet goes down while  you are presenting